Expensify/App
# Use encrypted auth token for images

### Details
Uses an encrypted auth token instead of a plain one in URLs to fetch images.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/166503

### Tests / QA Steps
1. Upload an image in a chat.
1. Verify that you can view the thumbnail image in the chat.
1. Tap on the image, and verify that you can view the full-size preview in the attachment modal.
1. Make sure you are not logged into E.com.
1. Upload a `.zip` file to chat.
1. Click on the attachment link for the `.zip` file, verify that it is downloaded without logging into E.com.
1. Log out then back in, then repeat steps 2, 3, & 6
1. Open the developer console, go to Application, search for the `session` key, and erase the key-value pair for `encryptedAuthToken`. Then refresh the page and repeat steps 2, 3, & 6.

### Tested On


### Screenshots

#### Web
![image](https://user-images.githubusercontent.com/47436092/122119796-7851e980-cdde-11eb-884c-1516ffdfa466.png)
![image](https://user-images.githubusercontent.com/47436092/122119907-9ddef300-cdde-11eb-9169-e04940554080.png)

#### Mobile Web


#### Desktop


#### iOS


#### Android
